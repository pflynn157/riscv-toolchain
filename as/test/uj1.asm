lui x10, 12345
auipc x11, 45678
jal x12, 4
