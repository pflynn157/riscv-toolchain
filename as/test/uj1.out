03039537
0b26e597
0040066f
